### PR TITLE
add fullmatch provider

### PR DESCRIPTION
**Pull request by salman731**
New provider (FullMatch) to watch full football matches

### CODE REVIEW

File: FullMatch/src/main/kotlin/com/redowan/FullMatchProvider.kt [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+        "/uefa-europa-league-24-25" to "UEFA Europa League 24/25",
+        "/uefa-nations-league" to "UEFA Nations League",
+    )
+
+    override suspend fun getMainPage(
+        page: Int,
+        request: MainPageRequest
+    ): HomePageResponse {
+
+        if(request.data == "/"){
```
Comment (redowan99) [inline FullMatch/src/main/kotlin/com/redowan/FullMatchProvider.kt:79]: Why are you using different method for latest post, when you just can use ".post-item"

**Comment by salman731** [comment]
It is using ajax request to load more posts that's why I used it..

**Comment by redowan99** [comment]
In load() you used tv is there are page TvSeries LoadResponse, is there are page where two videos present?

**Comment by salman731** [comment]
Yes, Some posts have two parts...

**Comment by redowan99** [comment]
Can you send me a link

**Comment by salman731** [comment]
https://fullmatch.info/2024/09/06/argentina-vs-chile-fifa-world-cup-2026-qualification-05-september-2024

**Pull request merged**